src/qt/addressbookpage.cpp

```diff
@@ -241,9 +241,7 @@ void AddressBookPage::on_showAddressQRCode_clicked()
 
```

Comment:
Hmm long time ago 🙈 but looks like its just because `QDialog` doesn't need it for anything, `OptionsModel`'s constructor changed, so i just dropped the related stuff it seems. Also a7024cf28cb910c17d71968c9b816286b6fa9f00 now.